web/files/files.go

```diff
@@ -707,7 +708,7 @@ func ThumbnailHandler(c echo.Context) error {
 		return jsonapi.NewError(http.StatusBadRequest, "Wrong download token")
 	}
 
-	fs := instance.ThumbsFS()
+	fs := lifecycle.ThumbsFS(instance)
```

Comment:
related to previous comments, should be `fs, err := ...` to handle error when constructing ThumbsFS name.